SwrveCommonSDK/src/main/java/com/swrve/sdk/config/SwrveConfigBase.java

```diff
@@ -121,6 +121,11 @@
     private int defaultBackgroundColor = Color.TRANSPARENT;
 
     /**
+     * HTTP timeout used when contacting the Swrve APIs.
+     */
+    private int httpTimeout = 60000;
+
```

Comment:
Might be better to use constants in the future (and longs), especially for milliseconds. Can be error prone. eg:
private static final long SECOND = 1000;
private static final long httpTimeout = SECOND*60;